index.js

```diff
@@ -0,0 +1,16 @@
+import express from "express";
+import db from "./src/config/dbConfig";
+
+const app = express();
+
+const PORT = process.env.PORT || 3000;
+
+app.listen(PORT, console.log(`Listening on port ${PORT}.......`));
```

Comment:
The second argument should be a function.

```js
app.listen(PORT, () => console.log(`Listening on port ${PORT}.......`));
```

Reply:
Alright, let me correct it